gazebo/physics/ode/ODEPhysics.cc

```diff
@@ -352,13 +353,16 @@ void ODEPhysics::Init()
 //////////////////////////////////////////////////
 void ODEPhysics::InitForThread()
 {
+  IGN_PROFILE_THREAD_NAME("ODEPhysics");
   dAllocateODEDataForThread(dAllocateMaskAll);
 }
 
 //////////////////////////////////////////////////
 void ODEPhysics::UpdateCollision()
 {
   DIAG_TIMER_START("ODEPhysics::UpdateCollision");
+  IGN_PROFILE("ODEPhysics:UpdateCollision");
+  IGN_PROFILE_BEGIN("dSpaceCollide");
```

Comment:
In other places of this PR, I see the the profile name prefixed by the class name. For example `AltimeterSensor::publish`. But that's also strange since it makes me think that `publish` is a function belonging `AltimeterSensor`, which is not the case. Looking at ign-gazebo, I see a convention like `class_name::function_name profile_name`, e.g. [here](https://github.com/ignitionrobotics/ign-gazebo/blob/master/src/rendering/RenderUtil.cc#L329). Maybe we can follow the same convention? Alternatively, just do what you have here now, i.e. `dSpaceCollide` without the class name prefix name, and update other places in the PR to keep them consistent.

Reply:
Fixed and added your feedback https://github.com/osrf/gazebo/pull/2776/commits/f8d068fb7400597ff0d21425063107bee714ade0